### PR TITLE
Update equipment.md

### PR DESCRIPTION
**Pull request by Misaki290**
Added some info and recommendations on wearing your equipment.
Corrected the colours for legendary and epic caches.

### CODE REVIEW

**Review by DexterHuang — COMMENTED** [review]
@Misaki290  I have commented something 0. 0

File: mobile-tutorial/equipment.md [hunk 1]
```diff
@@ -1,28 +1,52 @@
...
+  - High Quality (Green)
+  - Rare (Yellow)
+  - Legendary (Fuchsia, Deep Pink)
+  - Epic (Dark Red)
+
+Higher rarities will have better stats and more module upgrade slots, depending on the piece of equipment.  
+High level equipment will give better stats compared to lower level ones.  
+
+Completing the "Say Hello to the Global Chat" quest will offer a complete random level 1 gear.  
+
```
[the remark this answers is not in the document]
Comment (Misaki290) [reply mobile-tutorial/equipment.md:16]: Okay, got it. 👍

**Review by BruteDeity — COMMENTED** [review]
why are we using lvl and then 5 words later using level? shouldn't it be either or and not both?

**Comment by Misaki290** [comment]
> why are we using lvl and then 5 words later using level? shouldn't it be either or and not both?

I was tired when I was writing it and didn't put much through into it. Anyway, it was just at "Upgrade modules" part when I was giving an example. In other cases, I just used "level" exclusively.
Anyway, nice call. 😅

**Comment by BruteDeity** [comment]
Awesome thanks misaki! it all looks good!

**Pull request merged**